ttnn/ttnn/operations/others.py

```diff
@@ -505,4 +505,611 @@ def upsample(
     return output_tensor
 
 
+def _zeros_like_validate_input_tensors(operation_name, input_tensor, *args, **kwargs):
```

Comment:
the newly code should be removed from others.py and moved into creation.py, reduction.py and losses.py